blazor/chart/zooming.md

```diff
@@ -373,6 +373,101 @@ The axis interval will be calculated automatically with respect to the zoomed ra
 
 ![Auto Interval on Zooming in Blazor Area Chart](images/zoom/blazor-area-chart-auto-interval-zooming.png)
 
+## Zoom Toolbar Positioning
+
+Using `ChartZoomToolbarPosition`, you can customize the position of the zoom toolbar according to your preference. You can align it horizontally using the `HorizontalAlign` property with options such as `Right`, `Left`, and `Center`. Similarly, you can align it vertically using the `VerticalAlign` property with options including `Top`, `Bottom`, and `Middle`. Additionally, for more precise positioning, you can specify custom coordinates using `X` and `Y` values.
+
+```cshtml
```

Comment:
Check EJ2 https://helpej2.syncfusion.com/documentation/chart/zooming#toolbar-customization and follow the sample as in it.